setup.py

```diff
@@ -60,4 +60,8 @@
         'scipy == 0.17.1',
     ],
 
+    # pytest integration
+    setup_requires=['pytest-runner'],
```

Comment:
@dhimmel are you referring to the scipy=0.17.1 version or the pytest-runner not having an associated version? I think pytest-runner not having an associated version is the correct approach, as there we want to continue being able to get the latest stable version where possible.